chef-server-stats/chef-server-stats

```diff
@@ -295,6 +316,7 @@ output.merge!(get_authz_stats(services))
 output.merge!(get_redis_stats(services))
 output.merge!(get_server_status)
 output.merge!(get_folsom_metrics(chef_server_12))
+output.merge!(get_pooler_metrics(chef_server_12))
```

Comment:
I was debating that. Folsom could be interesting if you accept it as a long running snapshot of application performance, but relatively useless as a monitoring feed. I feel like we could safely remove it from here, as the prime audience for stats is a customer feeding this information into a TSDB that can crunch the stats collected into a histogram type dataset anyway (e.g. what was my average free pool members over the past 15, 60, 180 mins), but didn't want to make too many assumptions.